cli/src/command.rs

```diff
@@ -415,6 +420,17 @@ pub fn run() -> Result<()> {
 			#[cfg(not(feature = "polkadot-native"))]
 			panic!("No runtime feature (polkadot, kusama, westend, rococo) is enabled")
 		},
+		Some(Subcommand::HostPerfCheck(cmd)) => {
+			let mut builder = sc_cli::LoggerBuilder::new("info");
+			builder.with_colors(true);
+			let _ = builder.init();
```

Comment:
```suggestion
			builder.init()?;
```


No reason to swallow the error.